src/main/java/io/in_toto/keys/RSAKey.java

```diff
@@ -225,18 +219,21 @@ public String computeKeyId() {
     }
 
     private void encodePem(Writer out, boolean privateKey) {
+
         JcaPEMWriter pemWriter = new JcaPEMWriter(out);
+        
         try {
+			
             if (privateKey && getPrivate() != null)
-                pemWriter.writeObject(new MiscPEMGenerator(this.kpr.getPrivateKeyInfo()));
+				pemWriter.writeObject(new MiscPEMGenerator(this.kpr.getPrivateKeyInfo()));
             else
-                pemWriter.writeObject(new MiscPEMGenerator(this.kpr.getPublicKeyInfo()));
+				pemWriter.writeObject(new MiscPEMGenerator(this.kpr.getPublicKeyInfo()));
```

Comment:
we are using 4 spaces for tabs right now, although most guidelines suggest 2. Either way, we shouldn't be using tabs.

Reply:
done